### PR TITLE
feat(sdk): Add support for Magic Eden Wallet

### PR DESCRIPTION
**Pull request by danadi7**
#### Which issue(s) does this PR fixes?:

Fixes #

### CODE REVIEW

File: apps/browser-tests/src/App.tsx [hunk 1]
```diff
@@ -2,11 +2,14 @@ import { useCallback, useMemo, useState } from "react";
...
 import * as xverse from "@ordzaar/ordit-sdk/xverse";
+import * as magiceden from "@ordzaar/ordit-sdk/magiceden";
 
 import { RadioInput } from "./components/RadioInput";
 import { Select } from "./components/Select";
 
-type WalletProvider = "unisat" | "xverse";
+type WalletProvider = "unisat" | "xverse" | "magiceden";
+
+const network = "testnet" as const;
```
Comment (Nanosync) [inline apps/browser-tests/src/App.tsx:12]:
```suggestion
const TESTNET = "testnet" as const;
```

**Review by Nanosync — CHANGES_REQUESTED** [review]
Minor comment added. I think the base branch should be the refactor branch as there's duplicate code to review

File: packages/sdk/src/browser-wallets/magiceden/index.ts [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    getMagicEdenWalletProvider,
+    message,
+    address,
+    network,
+  );
+}
+
+export {
+  getAddresses,
+  getMagicEdenWalletProvider,
```
Comment (Nanosync) [inline packages/sdk/src/browser-wallets/magiceden/index.ts:108]: We can keep this function internal since it's not in the docs yet

```suggestion
```

**Review by Nanosync — DISMISSED** [review]

File: apps/browser-tests/src/App.tsx [hunk 3]
```diff
@@ -1,12 +1,20 @@
...
 import * as xverse from "@ordzaar/ordit-sdk/xverse";
 
 import { RadioInput } from "./components/RadioInput";
 import { Select } from "./components/Select";
 
-type WalletProvider = "unisat" | "xverse";
+type WalletProvider = "unisat" | "xverse" | "magiceden";
+
+const TESTNET = "testnet" as const;
+// const MAINNET = "mainnet" as const;
```
Comment (thedoublejay) [inline apps/browser-tests/src/App.tsx:13]: For another ticket @danadi7, can just add a button here instead of hard coding it. Not for this PR

**Review by thedoublejay — APPROVED** [review]

File: apps/browser-tests/src/App.tsx (hunk 3 above)
Comment (danadi7) [reply apps/browser-tests/src/App.tsx:13]: This was intentional, I didn't want to make it too easy to switch to `mainnet` for safety reasons just in case.

Comment (thedoublejay) [reply apps/browser-tests/src/App.tsx:13]: But this is only a testing app :D

**Pull request merged**